app/src/core/ffmpeg.ts

```diff
@@ -278,6 +278,20 @@ export async function convertToWav(
   return fileData;
 }
 
+export async function copyToMp4(input_path: string): Promise<Buffer> {
+  const tempdir: string = await getTempDir();
+  const outputFile = path.join(tempdir, 'output.mp4');
+
+  const cmd = new FFmpegCommand({ y: undefined });
```

Comment:
what does this y: undefined do?

Reply:
I don't know. I copied that from convertToWav

Reply:
Ahhhh, I see where that comes from. I think we also copied it to convertToWav from the ffmpeg exporter. This leads to fessonia settting the `-y` command line flag which allows ffmpeg to overwrite the output file if it already exists

Reply:
I don't care about this flag, should we remove it or keep it?

Reply:
Ah it is fine. I thought that undefined might be the wrong value (expected true here) but this is fine :)